offer-manager-domain/src/test/groovy/life/qbic/business/offers/QuantityDiscountSpec.groovy

```diff
@@ -19,7 +19,7 @@ class QuantityDiscountSpec extends Specification {
         def function = new QuantityDiscount()
         double result = function.apply(sampleCount, discountablePrice)
         then: "the discount is applied correctly"
-        result == discountMap.get(sampleCount) * discountablePrice
+        result == 1-discountMap.get(sampleCount) * discountablePrice
```

Comment:
```suggestion
        result == ( 1 - discountMap.get(sampleCount) )* discountablePrice
```
How does this work without the paranthesis?

Reply:
works very well, thank you!

...I have no idea, thanks for the suggestion